src/dataflow/src/render/join/delta_join.rs

```diff
@@ -180,6 +182,10 @@ impl DeltaJoinPlan {
             });
         }
 
+        // Now that `map_filter_project` has been capture in the state builder,
```

Comment:
nit: captured

Reply:
Will fix locally and push later, to avoid a CI re-run (is there a better way to do cosmetic updates w/o spending 20 minutes of compute?).